backend/app/adapter/google/account_integration_test.go

```diff
@@ -0,0 +1,91 @@
+package google
```

Comment:
Please add `// +build integration all` build tag.

All integration tests are run by this [script](https://github.com/short-d/short/blob/master/backend/scripts/integration-test)

Reply:
Please add the build tag @alldroll suggested. Build tag is one of the techniques we use to select which tests to run. We are separating unit tests and integration tests because when unit tests fail ( check whether the module itself is implemented correctly ), there is no point of running integration tests ( check whether two modules are interacting correctly). This allows us to find failures in the shortest time.

Reply:
addressed comment, thanks for letting me know